src/balances/entities/schemas/balance.schema.ts

```diff
@@ -0,0 +1,38 @@
+import { JSONSchemaType } from 'ajv';
+import { Balance } from '../balance.entity';
+import { TokenInfo } from '../../../common/entities/token-info.entity';
+import { TokenType } from '../../../common/entities/token-type.entity';
+
+const tokenInfoSchema: JSONSchemaType<TokenInfo> = {
+  type: 'object',
+  properties: {
+    tokenType: {
+      type: 'string',
+      enum: [
+        TokenType.Erc20,
+        TokenType.Erc721,
+        TokenType.NativeToken,
+        TokenType.Unknown,
+      ],
+    },
+    address: { type: 'string' },
+    decimals: { type: 'number' },
+    symbol: { type: 'string' },
+    name: { type: 'string' },
+    logoUri: { type: 'string', nullable: true },
+  },
+  required: [],
+};
```

Comment:
`TokenInfo` is part of common as it will be shared by different routes. The schema should live also in `/common` because other routes will end up depending on it too.